src/game/common/audio/audiomisc.cpp

```diff
@@ -66,6 +66,7 @@ FieldParse MiscAudio::s_fieldParseTable[] = { { "RadarNotifyUnitUnderAttackSound
     { "AircraftWheelScreech", &INI::Parse_Audio_Event_RTS, nullptr, offsetof(MiscAudio, m_aircraftWheelScreech) },
     { nullptr, nullptr, nullptr, 0 } };
 
+// Was originally INI::parseMiscAudio
```

Comment:
Why no definition for this one

Reply:
I don't know, ask EA :)
The original function in the game is named INI::parseMiscAudio.